plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioConnectorSmokeTest.java

```diff
@@ -0,0 +1,89 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.trino.plugin.hive.containers.HiveMinioDataLake;
+import io.trino.testing.QueryRunner;
+import io.trino.tpch.TpchTable;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+
+import static io.trino.plugin.hive.containers.HiveMinioDataLake.ACCESS_KEY;
+import static io.trino.plugin.hive.containers.HiveMinioDataLake.SECRET_KEY;
+import static io.trino.plugin.iceberg.IcebergQueryRunner.createIcebergQueryRunner;
+import static io.trino.testing.sql.TestTable.randomTableSuffix;
+import static java.lang.String.format;
+
+public class TestIcebergMinioConnectorSmokeTest
+        extends BaseIcebergConnectorSmokeTest
+{
+    private final String bucketName;
+
+    private HiveMinioDataLake hiveMinioDataLake;
+
+    public TestIcebergMinioConnectorSmokeTest()
+    {
+        super("tpch_" + randomTableSuffix());
+        this.bucketName = "test-iceberg-minio-integration-test-" + randomTableSuffix();
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        this.hiveMinioDataLake = closeAfterClass(new HiveMinioDataLake(bucketName, ImmutableMap.of()));
+        this.hiveMinioDataLake.start();
+        return createIcebergQueryRunner(
+                ImmutableMap.of(),
+                ImmutableMap.<String, String>builder()
+                        .putAll(ImmutableMap.<String, String>builder()
+                                .put("iceberg.catalog.type", "HIVE_METASTORE")
+                                .put("hive.metastore.uri", "thrift://" + hiveMinioDataLake.getHiveHadoop().getHiveMetastoreEndpoint())
+                                .put("hive.s3.aws-access-key", ACCESS_KEY)
+                                .put("hive.s3.aws-secret-key", SECRET_KEY)
+                                .put("hive.s3.endpoint", "http://" + hiveMinioDataLake.getMinio().getMinioApiEndpoint())
+                                .put("hive.s3.path-style-access", "true")
+                                .put("hive.s3.streaming.part-size", "5MB")
+                                .buildOrThrow())
+                        .buildOrThrow(),
+                SchemaInitializer.builder()
+                        .withSchemaName(schemaName)
+                        .withClonedTpchTables(
+                                ImmutableSet.<TpchTable<?>>builder()
+                                        .addAll(REQUIRED_TPCH_TABLES)
+                                        .build())
```

Comment:
No need to copy REQUIRED_TPCH_TABLES

Reply:
It's aligned with TestIcebergConnectorSmokeTest which share same base set of tests.
I've checked copied tables may be limited to Nation and Region.
Will update it in source test `TestIcebergConnectorSmokeTest` as well.